lib/ProteinStructureUtils/Utils/PDBUtils.py

```diff
@@ -185,8 +327,124 @@ def import_model_pdb_file(self, params):
+            returnVal.update(report_output)
+
+        return returnVal
+
+    def import_experiment_pdb_file(self, params, create_report=True):
+        """
+        import_experiment_pdb_file: upload an experiment pdb file and convert into a
+                                   KBaseStructure.ExperimentalProteinStructure object
+        """
+
+        file_path, workspace_name, mmcif_name = self._validate_import_pdb_file_params(params)
+
+        if not isinstance(workspace_name, int):
+            workspace_id = self.dfu.ws_name_to_id(workspace_name)
+        else:
+            workspace_id = workspace_name
+
+        # TODO: Figure out how to parse the experimental pdb file for an experimental data structure
+        data, n_polypeptides,  = self._exp_file_to_data(file_path)
+        data['user_data'] = params.get('description', '')
+        logging.info(data)
+
+        info = self.dfu.save_objects({
+            'id': workspace_id,
+            'objects': [
+                {'type': 'KBaseStructure.ExperimentalProteinStructure',
+                 'name': mmcif_name,
+                 'data': data}]
+        })[0]
+        obj_ref = f"{info[6]}/{info[0]}/{info[4]}"
+
+        returnVal = {'structure_obj_ref': obj_ref}
+
+        if create_report:
+            report_output = self._generate_report('import_experiment_pdb_file', obj_ref,
+                                                  workspace_name, n_polypeptides,
+                                                  mmcif_name, file_path)
+            returnVal.update(report_output)
+
+        return returnVal
+
+    def batch_import_pdb_files(self, params):
+        """
+        batch_import_pdb_files: upload two sets of pdb files and convert into a
+                               KBaseStructure.ProteinStructures object
+        1. from the params['meta_file_path'] spreadsheet, sort out the model_pdb_paths,
+           exp_pdb_paths and the kbase_meta_data
+        2. call import_model_pdb_file on each entry in model_pdb_paths,
+           call import_experiment_pdb_file on each entry in exp_pdb_paths, and
+           call _parse_kbase_metadata to parse for KBase metadata
+        3. assemble the data for a ProteinStructures and save the data object
+        4. generate a report
+        """
+
+        (model_pdb_files, exp_pdb_files, workspace_name,
+         structures_name) = self._validate_batch_import_pdb_files_params(params)
+
+        if not isinstance(workspace_name, int):
+            workspace_id = self.dfu.ws_name_to_id(workspace_name)
+        else:
+            workspace_id = workspace_name
+
+        model_pdb_objects = list()
+        exp_pdb_objects = list()
+        successful_files = list()
+        failed_files = list()
+        protein_structures = dict()
+        total_structures = 0
+
+        # loop through the lists model_pdb_file_paths and exp_pdb_file_paths
+        if model_pdb_files:
```

Comment:
you don't need this if clause if you init model_pdb_files as an empty list by default.